federatorai/manifest.json

```diff
@@ -0,0 +1,28 @@
+{
+  "display_name": "Federator.ai",
+  "maintainer": "support@prophetstor.com",
+  "manifest_version": "1.0.0",
+  "name": "federatorai",
+  "metric_prefix": "federatorai.",
+  "metric_to_check": "federatorai.integration.status",
+  "creates_events": false,
+  "short_description": "Integration with Federator.ai to accelerate/optimize application performance",
+  "guid": "ec0fd93a-ee4c-4652-9996-cc68cb5a4d45",
+  "support": "contrib",
+  "supported_os": [
+    "linux"
+  ],
+  "public_title": "Datadog-federatorai Integration",
+  "categories": [
+    ""
```

Comment:
this can be:
    "containers",
    "orchestration",

Reply:
thanks, i have added "containers","orchestration" to categories